src/test/java/uk/nhs/prm/deduction/e2e/models/EhrRequestMessage.java

```diff
@@ -0,0 +1,46 @@
+package uk.nhs.prm.deduction.e2e.models;
+
+import uk.nhs.prm.deduction.e2e.utility.Resources;
+
+import java.util.UUID;
+
+import static uk.nhs.prm.deduction.e2e.utility.TestUtils.getUUIDAsUpperCaseString;
+
+public class EhrRequestMessage {
+    private final String nhsNumber;
+    private final String sourceGpOds;
+    private final String destinationGpOds;
+    private final String sourceGpAsid;
+    private final String destinationGpAsid;
+    private final UUID conversationId;
+    private final UUID messageId;
+
+    public EhrRequestMessage(String nhsNumber, String sourceGpOds, String destinationGpOds, String sourceGpAsid, String destinationGpAsid, UUID conversationId, UUID messageId) {
+        this.nhsNumber = nhsNumber;
+        this.sourceGpOds = sourceGpOds;
+        this.destinationGpOds = destinationGpOds;
+        this.sourceGpAsid = sourceGpAsid;
+        this.destinationGpAsid = destinationGpAsid;
+        this.conversationId = conversationId;
+        this.messageId = messageId;
+    }
+
+    public String toJsonString() {
```

Comment:
Same as above again! :D 

```suggestion
    public String getJsonString() {
```